randovania/layout/preset_describer.py

```diff
@@ -1,70 +1,70 @@
 import collections
-from typing import Iterable, Sequence
+from collections.abc import Iterable, Sequence
 
 from randovania.game_description import default_database
 from randovania.game_description.pickup.standard_pickup import StandardPickupDefinition
 from randovania.generator.pickup_pool import pool_creator
 from randovania.layout.base.ammo_pickup_configuration import AmmoPickupConfiguration
 from randovania.layout.base.available_locations import RandomizationMode
 from randovania.layout.base.base_configuration import BaseConfiguration
+from randovania.layout.base.damage_strictness import LayoutDamageStrictness
 from randovania.layout.base.dock_rando_configuration import DockRandoMode
 from randovania.layout.base.standard_pickup_configuration import StandardPickupConfiguration
 from randovania.layout.base.pickup_model import PickupModelStyle
 from randovania.layout.preset import Preset
 
 
-def _bool_to_str(b: bool) -> str:
-    if b:
-        return "Yes"
-    else:
-        return "No"
-
-
 PresetDescription = tuple[str, list[str]]
 
 
 class GamePresetDescriber:
-    def _calculate_starting_items(self, configuration: BaseConfiguration) -> list[str]:
-        expected_count = self.expected_starting_item_count(configuration)
-        starting_items = []
+    def _calculate_pickup_pool(self, configuration: BaseConfiguration) -> list[str]:
```

Comment:
it might be worthwhile to split the pool size into major and minor for the preset describer as well, when it's active

Reply:
oh I plan to do that. Thinking of changing `pool_creator.calculate_pool_pickup_count` to always return the two values for each `LocationCategory`.